thunder/tests/test_jit_general.py

```diff
@@ -631,6 +631,7 @@ def test_nanogpt():
     "device",
     ("cpu", "cuda"),
 )
+@pytest.mark.large_mem
```

Comment:
There are some nicer alternative solutions that we could pursue:

- Make the "like" configs smaller.
- Run these tests on 16bit precision.

For reference, this is how much the params take today:
```
gpt-neox-like 6532864 26.131456 MB
llama1-like 602432 2.409728 MB
long-context-like 4301120 17.20448 MB
llama2-like 602432 2.409728 MB
falcon-7b-like 4359040 17.43616 MB
falcon-40b-like 1589760 6.35904 MB
codellama2-like 819520 3.27808 MB
mixtral-like 3308800 13.2352 MB
```

(in litgpt_model.py)
```python
for name, config in name_to_config.items():
    if not name.endswith("-like"):
        continue
    config = Config(**config)
    with torch.device("meta"):
        model = GPT(config)
    n_params = sum(p.numel() for p in model.parameters())
    print(name, n_params, (n_params * 4) / 1e6, "MB")
```

Reply:
These reported numbers look small. Maybe there should be a better clean-up after each test?